cilium-health/launch/endpoint.go

```diff
@@ -57,7 +57,11 @@ const (
 var (
 	// vethName is the host-side veth link device name for cilium-health EP
 	// (veth mode only).
-	vethName = "cilium_health"
+	vethName = "lxcciliumhealth"
```

Comment:
Small nit: maybe just `lxc_health` which would fit into the name limit

Reply:
Yeah, sounds better than `lxcciliumhealth`. Will change once the CI build has finished